app/errors.py

```diff
@@ -0,0 +1,16 @@
+class VaccineError(Exception):
+    def __str__(self) -> str:
+        return f"{self.__class__.__name__}: Person is not vaccinated"
+
+
+class NotVaccinatedError(VaccineError):
+    """Person is not vaccinated"""
+
+
+class OutdatedVaccineError(VaccineError):
+    """The vaccine term has expired"""
+
+
+class NotWearingMaskError(Exception):
+    def __str__(self) -> str:
+        return f"{self.__class__.__name__}: Person is not wearing a mask"
```

Comment:
There is no need to add a class name in the string method, the exception class would be seen in a traceback in case raised. Please, remove it. Besides that, it would be better to avoid linking string descriptions to a particular class or name to which is related, e.g. `Person`. Keep the error string more indicative of the exception itself so as not to limit it to Person. Maybe, it would relate to animals in the future as well